src/web/pages/alerts/component.jsx

```diff
@@ -681,6 +720,7 @@ class AlertComponent extends React.Component {
 
           const result_filter_id = selectSaveId(result_filters);
           const report_format_id = selectSaveId(report_formats);
+          const report_config_id = selectSaveId(report_configs);
```

Comment:
Perhaps `UNSET_VALUE` could be passed here so that we don't get the first report config in the list. 

I noticed when running locally that if I open `New Alert` dialog and change the `Method` to `SCP` for example, I see `Report Config` displaying the first value in the list of the report configs even though it does not match the selected `Report Format`.